*** ./smali/com/android/server/SystemServer.smali	2017-08-18 18:18:17.406854771 +1000
--- /home/sostk/miui8/kyleprods/temp/new_smali/services.jar.out/./smali/com/android/server/SystemServer.smali	2017-08-18 18:17:03.730850944 +1000
***************
*** 142,147 ****
--- 142,149 ----
      .param p0, "args"    # [Ljava/lang/String;
  
      .prologue
+     invoke-static {}, Lmiui/patchrom/ClassHook;->initServerHook()V
+ 
      new-instance v0, Lcom/android/server/SystemServer;
  
      invoke-direct {v0}, Lcom/android/server/SystemServer;-><init>()V
